_security-analytics/sec-analytics-config/index.md

```diff
@@ -12,9 +12,10 @@ redirect_from:
 
 Before Security Analytics can begin generating findings and sending alerts, administrators must create detectors and make log data available to the system. Once detectors are able to generate findings, you can fine-tune your alerts to focus on specific areas of interest. The following steps outline the basic workflow for setting up components in Security Analytics.
 
-1. Create security detectors and alerts, and ingest log data. See [Creating detectors]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/) for details.
+1. Create threat detectors and alerts, and ingest log data. See [Creating detectors]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/) for details.
+1. Consider [creating correlation rules]({{site.url}}{{site.baseurl}}/sec-analytics-config/correlation-config/) to identify connections between events and possible threats occurring in different logs and throughout your system.
```

Comment:
```suggestion
1. Consider [creating correlation rules]({{site.url}}{{site.baseurl}}/sec-analytics-config/correlation-config/) to identify connections between events and possible threats occurring in different logs or throughout your system.
```

Reply:
Even better to get rid of the conjunction altogether. Separate parts of the system generate their own logs.